rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -91,6 +93,95 @@ object Jupyter {
                             Some(xbounds))
     })
 
+  def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
+    val labels = samples.head.keys.toList.sorted
+    val data = labels.map { k =>
+      k -> samples.map { m =>
+        m(k)
+      }
+    }
+    whiskers(data, None)
+  }
+
+  def whiskers[K, N](seq: Seq[(K, Seq[N])], ybounds: Option[Bounds])(
+      implicit num: Numeric[N],
+      theme: Theme): Plot = {
+    val data = seq.map { case (_, ns)  => ns.map(num.toDouble) }
+    val labels = seq.map { case (k, _) => k.toString }
+
+    val xb = Bounds(0, seq.size.toDouble)
+    val yb = ybounds.getOrElse(
+      Bounds(
+        data.flatten.reduceOption[Double](math.min).getOrElse(0),
+        data.flatten.reduceOption[Double](math.max).getOrElse(0)
+      ))
+
+    val boxContexts = data.zipWithIndex.map {
+      case (dist, index) =>
+        if (dist.nonEmpty) {
+          val summary = BoxPlotSummaryStatistics(dist, (0.25, 0.50, 0.75))
+          Some(BoxRenderer.BoxRendererContext(summary, index))
+        } else None
+    }
+
+    Plot(
+      xb,
+      yb,
+      BoxPlotRenderer(
+        boxContexts,
+        BoxRenderer.custom { (extent, ctx) =>
+          val stats = ctx.summaryStatistics
+          val mu = stats.allPoints.sum / stats.allPoints.size
+          val (low, high) = hdpi(stats.allPoints)
```

Comment:
I think you want stats.{lowerQuantile,upperQuantile} here instead of hdpi. If we want different quantiles we should probably make that a parameter.

Reply:
that's fair. I do want to make the default quantiles match the book's HDPI range. But it doesn't have to be the hdpi algo.